src/machine/operation.c

```diff
@@ -51,27 +42,39 @@ static int operation_done(sd_event_source *s, const siginfo_t *si, void *userdat
 
         o->pid = 0;
 
+        r = read_operation_errno(se, o);
+        if (r < 0)
+                log_debug_errno(r, "Operation failed: %m");
+
+        /* If a completion routine (o->done) is set for this operation, call it. It sends a response, but can return a error in which case it expect us to reply. */
```

Comment:
`can return a error` -> `can return an error`

Please drop the trailing `*/` and merge with the next line, and wrap about ~109ch.